test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js

```diff
@@ -172,6 +172,11 @@ describe('Navigation', () => {
       cy.get('button[title="Settings"]').should('contain', '');
     });
 
+    it('Icon with label label in TopNav', () => {
+      cy.get('button[title="Git"]>.fd-top-nav__icon').should('exist');
+      cy.get('button[title="Git"]').should('contain', 'Git');
```

Comment:
Just a small change, since we invented those nice test ids for the of e2e tests. I know, we've not refactored not all previous usages.
```suggestion
      cy.get('button[data-testid="icon-and-label"]>.fd-top-nav__icon').should('exist');
      cy.get('button[data-testid="icon-and-label"]').should('contain', 'Git');
```

Reply:
Added to new commit :)